### PR TITLE
Add tests for models.

### LINKED ISSUE #41: Tests for Models

### PR DESCRIPTION
**Pull request by damonkelley**
Resolves #41

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @somexpert @ldko

**Comment by somexpert** [comment]
:+1:

File: tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+class TestAgent:
+
+    def test_unicode(self):
+        agent = factories.AgentFactory.build()
+        assert agent.agent_name == unicode(agent)
+
+    def test_get_absolute_url(self):
+        agent = factories.AgentFactory.build()
+        url = reverse(
```
Comment (ldko) [inline tests/test_models.py]: This line is basically the exact code of the method this test is testing. Should we handle this test differently?

Comment (damonkelley) [inline tests/test_models.py]: Probably would hurt to do just stick the URI in there instead of using `reverse`

**Comment by ldko** [comment]
:+1:

**Pull request merged**